### DISCUSSION TITLE
Build-a-Mine: mission string

### DISCUSSION THREAD

**Discussion by alexrovw** [endless-sky/endless-sky#9604] [Ideas]
Hi everyone. I'm planning a mission string, and I would like to request your feedback. The idea is not fully developed, but this post will be updated as it matures.

_(All names and places are subject to change)_

### Build-a-Mine (WIP)

![Investments](https://github.com/endless-sky/endless-sky/assets/98854112/233a2b48-93bd-4778-a0d3-d686618520ab)

FirstLast is approached by an entrepreneur named Johann Lusk, initially for a simple passenger transport job. He is seeking passage to Ada (Aldebaran) from anywhere in the Deep to pitch a business plan to some potential investors. After the meeting goes south, he asks the player to take him back home to Rand (Zeta Aquilae). During the trip, he rants about those foolish investors and that his business idea is a sure thing.

Later, First Last hears from Lusk again as he asks the player to invest in his business idea. There is an abandoned mine on Clink that is for sale at a very cheap price. He has run spectrum analysis scans on the area and found that there is a rich supply of emeralds in an unexplored section of the natural cave system that the mine connects with. Emeralds, you see, are always in demand, as their unique chemical structure is integral to a refinement subprocess within the greater manufacturing process of hyperdrives. With some seed money and some elbow grease, he can produce a fortune.

After duping the player into investing in this boondoggle, Lusk learns that he wasn't as smart as he thought he was; the emerald supply was tainted with impurities, making them costly to refine and basically worthless.

Later still, Lusk has another bright idea. He's going to build his own mine! With blackjack! And hoo... no, actually just a mine. This time, the investment will pay off for the player, but they will have to go on an adventure, wheeling and dealing with pirates, government authorities, and Lusk himself. Reward options are TBD as well, but ballpark is provided in the flow chart.

**Comment by Zitchas** [comment]
Looks like a solid plan. I like it, overall. The salary probably needs to be reduced somewhat, as this would be a fairly early game missions set, and for that time period the amounts here are very significant.

That aside, the story itself looks good, with some interesting potential variations in it. This is exactly the sort of long (non-campaign) mission strings that we need a lot more of. Nice work!
